frontend/coprs_frontend/tests/test_commands/test_rawhide_to_release.py

```diff
@@ -144,17 +144,17 @@ def test_branch_fedora(self, capsys):
             "Handling builds in copr 'user2/barcopr', chroot 'fedora-rawhide-i386'",
             "  Fresh new build chroots: 1, regenerate 0",
             "Handling builds in copr 'user1/foocopr', chroot 'fedora-rawhide-x86_64'",
-            "Createrepo for copr 'user1/foocopr', chroot 'fedora-19-x86_64'",
+            "Createrepo for 'user1/foocopr', chroot 'fedora-19-x86_64'",
             "fedora-19-i386 - already exists.",
             "fedora-19-x86_64 - already exists.",
             "Handling builds in copr 'user2/barcopr', chroot 'fedora-rawhide-i386'",
             "Handling builds in copr 'user1/foocopr', chroot 'fedora-rawhide-x86_64'",
-            "Createrepo for copr 'user1/foocopr', chroot 'fedora-19-x86_64'",
+            "Createrepo for 'user1/foocopr', chroot 'fedora-19-x86_64'",
             "fedora-19-i386 - already exists.",
             "fedora-19-x86_64 - already exists.",
             "Handling builds in copr 'user2/barcopr', chroot 'fedora-rawhide-i386'",
             "  Fresh new build chroots: 0, regenerate 1",
             "Handling builds in copr 'user1/foocopr', chroot 'fedora-rawhide-x86_64'",
-            "Createrepo for copr 'user1/foocopr', chroot 'fedora-19-x86_64'",
+            "Createrepo for 'user1/foocopr', chroot 'fedora-19-x86_64'",
```

Comment:
Would you mind editing the `test_branch_fedora` so it tests some custom CoprDir, too?